### PR TITLE
DOC: Add comparison with Stata page to documentation

### PR DESCRIPTION
**Pull request by kylebarron**
- [ ] closes #####
- [ ] tests added / passed
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

I noticed that the pandas documentation has pages for [Comparison with SAS](https://pandas.pydata.org/pandas-docs/stable/comparison_with_sas.html), [Comparison with R](https://pandas.pydata.org/pandas-docs/stable/comparison_with_r.html) and [Comparison with SQL](https://pandas.pydata.org/pandas-docs/stable/comparison_with_sql.html), but nothing for newcomers from Stata.

This is heavily based off of the "Comparison with SAS" page, with all the examples and discussion fixed for Stata.

### CODE REVIEW

**Comment by kylebarron** [comment]
When I recompile with `python make.py html`, the command line interface still asks me
> Running Sphinx v1.7.1
> I'm about to DELETE the following:
> ['comparison_with_stata.rst']
>
> WARNING: I'd like to delete those to speed up processing (yes/no)? 

Is there a configuration file somewhere that I should add this document to?

**Comment by jreback** [comment]
you need to add to the ``pandas/doc/source/index.rst.template``

File: doc/source/comparison_with_stata.rst [hunk 1]
```diff
@@ -0,0 +1,679 @@
+.. currentmodule:: pandas
+.. _compare_with_stata:
+
+Comparison with Stata
+********************
```
Comment (jreback) [inline doc/source/comparison_with_stata.rst]: sections should be excatly underlined (otherwise you get warnings)

**Review by jreback — CHANGES_REQUESTED** [review]

**Comment by jreback** [comment]
can you render and post this section

**Comment by kylebarron** [comment]
I rendered it with `python make.py html`, and it looks fine, but Github doesn't let me attach an html file to an issue. How should I post it?

It seems like the Travis build failed from the time limit?
> The job exceeded the maximum time limit for jobs, and has been terminated.

https://travis-ci.org/pandas-dev/pandas/jobs/347824581#L2729

**Comment by kylebarron** [comment]
@jreback What can I do to get this merged?

**Comment by kylebarron** [comment]
The CircleCI build is failing on two TSPlot tests, which seem to be unrelated to this PR.

**Review by TomAugspurger — APPROVED** [review]
This looks really nice. I merged in master, which should fix the circle CI issues. ping on green.

**Comment by kylebarron** [comment]
Ping!

**Pull request merged**